app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css?family=Pacifico|Pontano+Sans" rel="stylesheet">
+  <title><%= content_for?(:title) ? yield(:title) : "Sweetsy!" %></title>
+  <%= stylesheet_link_tag    "application" %>
+  <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+
+<header class="nav-bar">
```

Comment:
Your `header` element is **not** inside the `body` element.   All content on the page should be in the `body`.  Review the format of an HTML document.